Docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.9"
```

Comment:
I think this file should have filename written in lowercase as `docker-compose.yml` is a default value in [official documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#service-configuration-reference).

Reply:
Done 😄